src/sider/dialogmoter/components/referat/Referat.tsx

```diff
@@ -252,12 +247,10 @@ const Referat = ({ dialogmote, mode }: ReferatProps): ReactElement => {
   });
 
   useEffect(() => {
-    if (currentNarmesteLederNavn) {
-      reset({
-        naermesteLeder: currentNarmesteLederNavn,
-      });
-    }
-  }, [reset, currentNarmesteLederNavn]);
+    reset({
```

Comment:
Dette er måten å sette default value fra async kall (ref https://github.com/orgs/react-hook-form/discussions/9046). Problemet her var at vi alltid satte verdien til `currentNarmesteLederNavn` fra `isnarmesteleder` i stedet for å bruke `initialValues` som henter verdi fra mellomlagret referat hvis det finnes.